packages/drivers/local-driver/src/localSessionStorageDb.ts

```diff
@@ -27,6 +27,9 @@ class LocalSessionStorageCollection<T> implements ICollection<T> {
 	public async distinct(key: any, query: any): Promise<any> {
 		throw new Error("Method Not Implemented");
 	}
+	public async findAndUpdate(query: any, value: T): Promise<{ value: T; existing: boolean }> {
+		throw new Error("Method not implemented.");
+	}
```

Comment:
@dannimad Try applying this change to your server upgrade PR. That should get past the build failure you're seeing right now.